SECURITY.md

```diff
@@ -0,0 +1,74 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd.,
+including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape)
+and any of it's [maintained plugins](https://github.com/ApeWorX?q=%22ape-%22&type=public&language=&sort=).
```

Comment:
I don't really have a better answer yet, nor is this a huge deal, but this query is not 100% right. It shows a few other repos besides plugins in it.

Reply:
Still could be improved https://github.com/search?q=org%3Aapeworx+ape-&type=repositories

Reply:
For some reason, the docs site repo shows in that list

Reply:
Found a better one